sdk-api-src/content/fileapi/nf-fileapi-createfile3.md

```diff
@@ -133,7 +133,7 @@ If the function fails, the return value is **INVALID_HANDLE_VALUE**. To get exte
 
 ## -remarks
 
-**CreateFile3** behaves exactly the same way as [CreateFile2](nf-fileapi-createfile2.md) with one exception; operations will fail if *lpFileName* is redirected via a reparse point or symbolic link. This behavior can be disabled with a new flag that can be added to the *dwFileFlags*.
+**CreateFile3** behaves exactly the same way as [CreateFile2](nf-fileapi-createfile2.md) with one exception; operations can fail if *lpFileName* is redirected via a reparse point or symbolic link. This behavior can be enabled with a new flag that can be added to the *dwFileFlags*.
```

Comment:
Would this be clearer, rather than referring to the behavior as "enabled" or "disabled":
```suggestion
**CreateFile3** behaves exactly the same way as [CreateFile2](nf-fileapi-createfile2.md) with one exception; operations can fail if *lpFileName* is redirected via a reparse point or symbolic link. Redirects can be disallowed by adding the **FILE_FLAG_DISALLOW_PATH_REDIRECTS** flag to the *dwFileFlags*.
```